diff --git a/niri-config/src/binds.rs b/niri-config/src/binds.rs
index c5b4e6f..fcfc8cc 100644
--- a/niri-config/src/binds.rs
+++ b/niri-config/src/binds.rs
@@ -187,10 +187,8 @@ pub enum Action {
     CenterWindowById(u64),
     CenterVisibleColumns,
     FocusWorkspaceDown,
-    #[knuffel(skip)]
     FocusWorkspaceDownUnderMouse,
     FocusWorkspaceUp,
-    #[knuffel(skip)]
     FocusWorkspaceUpUnderMouse,
     FocusWorkspace(#[knuffel(argument)] WorkspaceReference),
     FocusWorkspacePrevious,
